build-artifacts/project-template-gradle/src/main/java/com/tns/RuntimeHelper.java

```diff
@@ -104,13 +104,15 @@ public static Runtime initRuntime(Application app)
 					logger.write("Error while getting current proxy thumb");
 				e.printStackTrace();
 			}
-			ThreadScheduler workThreadScheduler = new WorkThreadScheduler(new Handler(Looper.getMainLooper()));
+			ThreadScheduler workThreadScheduler = new WorkThreadScheduler();
 			Configuration config = new Configuration(workThreadScheduler, logger, debugger, appName, null, rootDir,
 					appDir, classLoader, dexDir, dexThumb, v8Config);
-			runtime = new Runtime(config);
+
+			runtime = Runtime.initializeRuntimeWithConfiguration(config);
```

Comment:
I know this is merged but still

Can we rename these to Runtime.init() and Runtime.initWithConfiguration() ?

@Plamen5kov